notes/2024/2024-07-25.md

```diff
@@ -0,0 +1,88 @@
+# 2024-07-25 ESLint TSC Meeting Notes
+
+## Transcript
+
+[`2024-07-25-transcript.md`](2024-07-25-transcript.md)
+
+## Attending
+
+- Nicholas C. Zakas (@nzakas) - TSC
+- Milos Djermanovic (@mdjermanovic) - TSC
+- Francesco Trotta (@fasttime) - TSC
+
+@nzakas moderated, and @sam3k took notes.
+
+## Topics
+
+### Statuses
+
+* **@nzakas:**  has been working on `@eslint/json`, `@eslint/markdown`, the new config loading RFC, and started work on the core rewrite RFC (but that will take a while).
+
+* **@mdjermanovic:** finished `eslint-plugin-react` support for ESLint v9, and was mostly reviewing PRs.
+
+* **@fasttime:** finished updating `@types/eslint` to v9 and I've been also mostly busy reviewing issues and PRs.
```

Comment:
Maybe like this since we are using the 3rd person form in these comments?

```suggestion
* **@fasttime:** finished updating `@types/eslint` to v9 and has been also mostly busy reviewing issues and PRs.
```